gtkm/auth/service_auth.py

```diff
@@ -11,10 +10,12 @@
 from .database.database_operations import get_user, get_user_by_github_login, create_user
 from .sessions.session import backend, cookie, verifier
 from .sessions.session_data import SessionData
+from . import credentials
 
 Base.metadata.create_all(bind=engine)
 auth = APIRouter()
 
+credentials.add_to_env()
```

Comment:
Nie tak to powinno wyglądać.

Zamiast pliku `credentials.py` powinien być po prostu plik `.env` taki jak tu: https://github.com/sjopinie/sjopinie/tree/master/sjopinie

A w ramach kodu nie powinieneś ingerować w environmenta.